src/qt/transactionview.cpp

```diff
@@ -46,6 +46,11 @@ TransactionView::TransactionView(QWidget *parent) :
     hlayout->addSpacing(23);
 #endif
 
+    // Export button exposes CSV export functionality
+    QPushButton *exportButton = new QPushButton(tr("&Export"), this);
+    exportButton->setToolTip(tr("Export the data in the current tab to a file"));
```

Comment:
You might considering saying 'to a CSV file' instead here to be more specific if you have the room in the dialogue.